ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2625,6 +2625,40 @@ def test_numpy_instance_abs__(
     )
 
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="__invert__",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+        min_num_dims=0,
```

Comment:
you've set `min_num_dims=0` here hence scalar values will also be generated. Whereas, the error indicates that the minimum no of dimensions should be 2.

Reply:
got it